packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -123,19 +126,45 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
   void didChangeDependencies() {
     super.didChangeDependencies();
     if (!initController()) return;
-    unawaited(controller.startRecording());
-
-    cancelListeners();
+    try {
```

Comment:
instead of doing this logic in the screen, we want to do this in the screen's controller `NetworkController`. Checkout the guide in the dart doc for OfflineScreenControllerMixin. This provides pretty thorough documentation on how offline support for a screen should be set up: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/offline_data.dart/#L66-L124.

You can also use the ProfilerScreenController as an example: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/profiler/profiler_screen_controller.dart/#L81

Reply:
not sure how I missed it, apologies. I have fixed it now and scanning through if I have missed anything.